backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/radiobutton/BpkRadioButton.kt

```diff
@@ -107,8 +107,8 @@ private fun BpkRadioButtonImpl(
   selected: Boolean,
   onClick: (() -> Unit)?,
   modifier: Modifier = Modifier,
-  enabled: Boolean = true,
-  interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
```

Comment:
No need to create default params for internal methods if it's expected to be used with passing-though params. I may lead to forgetting pass param and potential errors in the future.